Lib/ftplib.py

```diff
@@ -133,13 +132,13 @@ def __exit__(self, *args):
                     self.close()
 
     def connect(self, host='', port=0, timeout=-999, source_address=None):
-        '''Connect to host.  Arguments are:
+        """Connect to host.  Arguments are:
          - host: hostname to connect to (string, default previous host)
          - port: port to connect to (integer, default previous port)
          - timeout: the timeout to set against the ftp socket(s)
          - source_address: a 2-tuple (host, port) for the socket to bind
            to as its source address before connecting.
-        '''
+        """
```

Comment:
Here's another example of something that's exclusively a formatting change, this isn't needed.